database/migrations/2016_03_22_090010_create_products_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateProductsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name');
+            $table->string('keywords');
+            $table->integer('quantity')->unsigned();
+            $table->decimal('price');
+            $table->string('description');
+            $table->string('image');
+            $table->integer('categories_id')->unsigned();
```

Comment:
foreign key should be named in single not plural, `category_id`, not `categories_id`